spec/requests/students_spec.rb

```diff
@@ -1,7 +1,31 @@
 require 'rails_helper'
 
-RSpec.describe "Students", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'Students', type: :request do
+  describe 'GET /index' do
+    context 'when there are no students' do
+      it 'responds with 200 status' do
+        get '/students'
+        expect(response).to have_http_status(:ok)
```

Comment:
🎱  Probably worth including an assertion that it returns content type JSON:

```rb
it 'successfully responds with json' do
  get students_path
        
  expect(response.content_type).to eq('application/json')
  expect(response).to have_http_status(:ok)
end
```

Reply:
Done, had to change `eq` to `include` because utf-8